okapi-core/pom.xml

```diff
@@ -84,6 +84,10 @@
       <groupId>com.cronutils</groupId>
       <artifactId>cron-utils</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.ongres.scram</groupId>
+      <artifactId>client</artifactId>
+    </dependency>
```

Comment:
Can you add a test to https://github.com/folio-org/okapi/blob/master/okapi-core/src/test/java/org/folio/okapi/service/impl/PostgresHandleTest.java that shows that SCRAM-SHA-256 actually works?

Reply:
> Can you add a test to https://github.com/folio-org/okapi/blob/master/okapi-core/src/test/java/org/folio/okapi/service/impl/PostgresHandleTest.java that shows that SCRAM-SHA-256 actually works?

Hoped you never asked :-)